civictechprojects/static/css/partials/_LogInController.scss

```diff
@@ -9,6 +9,10 @@
   overflow: auto;
 }
 
+.LogInController-root b {
```

Comment:
Rather than declare this here, I'd suggest removing this and the `font-weight: 300;` declaration on line 4. That weight is inconsistent with the rest of our text (typically 400) and is the root cause of why bold didn't appear bold before.

I had to look this up to figure it out; [relative font weighting](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#meaning_of_relative_weights) - which we use - doesn't really do what's expected with a base of 300 weight.